src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeControllerTest.java

```diff
@@ -157,11 +158,14 @@ public void setup() {
             leaseMetaDataChecker
         );
 
+        OcrValidationRetryManager ocrValidationRetryManager = new OcrValidationRetryManager(2, 300);
```

Comment:
shouldnt this be mocked/autowired like all other dependencies?

Reply:
autowired